src/service/iframe-transport-responses-impl.js

```diff
@@ -0,0 +1,55 @@
+/**
```

Comment:
Ignore changes to this file - it was changed in #10714 and will be rebased away.